client/style.css

```diff
@@ -1,66 +1,186 @@
-footer{
```

Comment:
All css looks really cool. Looks just like google

Reply:
Ngl I spent far too long trying to ape it 😂